contracts/src/ECDSA.sol

```diff
@@ -26,12 +26,10 @@ library ECDSA {
     function recover(bytes32 hash, bytes memory signature)
         internal
         pure
-        returns (address)
+        returns (address signer)
```

Comment:
I'm not sure I like this syntax. Golang does it too and I always try to avoid it.